client/my-sites/domains/domain-management/components/icann-verification/icann-verification-card.jsx

```diff
@@ -68,6 +68,7 @@ class IcannVerificationCard extends React.Component {
 			<EmailVerificationCard
 				changeEmailHref={ changeEmailHref }
 				contactEmail={ contactDetails.email }
+				errorMessage={ translate( 'Unable to resend ICANN verification email.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Resend ICANN Verification Email' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).